File: app/src/main/java/co/tpcreative/supersafe/common/response/DataResponse.java
package co.tpcreative.supersafe.common.response;
import java.io.Serializable;
import java.util.List;

import co.tpcreative.supersafe.common.entities.ItemEntity;
import co.tpcreative.supersafe.common.entities.MainCategoryEntity;
import co.tpcreative.supersafe.model.Authorization;
import co.tpcreative.supersafe.model.EmailToken;
import co.tpcreative.supersafe.model.ItemModel;
import co.tpcreative.supersafe.model.MainCategoryModel;
import co.tpcreative.supersafe.model.Premium;
import co.tpcreative.supersafe.model.User;

public class DataResponse  implements Serializable  {
    /*User*/
    public User user;
    /*Main category*/
    public List<MainCategoryModel> categoriesList;
    public MainCategoryModel category;
    /*Items*/
    public List<ItemModel>itemsList;

    /*user cloud*/
    public UserCloudResponse userCloud;

    /*RequestCode*/
    public RequestCodeResponse requestCode;

    public Authorization author;

    public Premium premium;
    public EmailToken email_token;

    public String nextPage;
}

File: app/src/main/java/co/tpcreative/supersafe/common/response/DataResponse.kt
package co.tpcreative.supersafe.common.response
import co.tpcreative.supersafe.model.*
import java.io.Serializable

class DataResponse : Serializable {
    /*User*/
    var user: User? = null

    /*Main category*/
    var categoriesList: MutableList<MainCategoryModel?>? = null
    var category: MainCategoryModel? = null

    /*Items*/
    var itemsList: MutableList<ItemModel?>? = null

    /*user cloud*/
    var userCloud: UserCloudResponse? = null

    /*RequestCode*/
    var requestCode: RequestCodeResponse? = null
    var author: Authorization? = null
    var premium: Premium? = null
    var email_token: EmailToken? = null
    var syncData: SyncData? = null
    var nextPage: String? = null
}